File: src/Python/easy/math/problem66.py
from typing import List

class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        return [int(d) for d in str((int("".join(str(digit) for digit in digits)) + 1))]


digits = [1,2,3]

solution = Solution()
solution.plusOne(digits)

File: src/JavaScript/easy/math/problem66.js
/**
 * @param {number[]} digits
 * @return {number[]}
 */


var plusOne = function(digits) {
    return ((BigInt(digits.join('')) + 1n).toString().split(-1,1))[0].split("").map(x => x * 1)
};

// let digits = [1,2,3]
// let digits = [9]
let digits = [6,1,4,5,3,9,0,1,9,5,1,8,6,7,0,5,5,4,3]

plusOne(digits)